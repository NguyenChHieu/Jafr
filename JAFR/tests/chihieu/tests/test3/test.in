1
12
20
1
6