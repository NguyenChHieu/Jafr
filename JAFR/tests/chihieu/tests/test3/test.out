Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!
- Study floating point representations by 31/08/23
- Study processes by 02/09/23
- Write an essay object-relational impedance by 02/09/23
- Learn set notation by 01/09/23

You have the following meetings today!
- ELEC1601 group meeting at 11:30

You have the following meetings scheduled over the next week!
- Michael Mai's welcome party on 31/08/23 at 18:00
- Flipping burgers development workshop on 31/08/23 at 08:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Which task(s) would you like to mark as completed?
1. Study floating point representations by 31/08/23
2. Study processes by 02/09/23
3. Practise UML by 03/09/23
4. Write an essay object-relational impedance by 02/09/23
5. Learn set notation by 01/09/23
Invalid task number(s) entered. Please enter valid task number(s).
Invalid task number(s) entered. Please enter valid task number(s).
Marked as complete.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
